editor/plugins/animation_state_machine_editor.h

```diff
@@ -188,4 +234,26 @@ class AnimationNodeStateMachineEditor : public AnimationTreeNodeEditorPlugin {
 	AnimationNodeStateMachineEditor();
 };
 
+class MultiTransitionEdit : public RefCounted {
```

Comment:
May be better to rename to `EditorAnimationMultiTransitionEdit` for better context

Reply:
Renamed.